### DISCUSSION TITLE
What is the best way to implement a A-Z vertical search?

### DISCUSSION THREAD

**Discussion by ewongchla** [microsoft-search/pnp-modern-search#3986] [Ideas]
It is very common for Search pages using A-Z list to filter the search results. Is there a way to implement that with PnP Modern Search with SPO?

Thanks!

**Comment by HNep** [comment]
Ive followed https://aequos-solutions.github.io/modern-data-visualizer/getting_started/tutorials/build_people_directory_graph/ but not the last part ive put a to z for each a separate page in sitepages and open on same page configure each page to show a* etc
